client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -262,10 +265,24 @@ const PluginsBrowser = React.createClass( {
 			return;
 		}
 
+		const suggestedSearches = [
+			this.props.translate( 'Engagement', { context: 'Plugins suggested search term' } ),
+			this.props.translate( 'Security', { context: 'Plugins suggested search term' } ),
+			this.props.translate( 'Appearance', { context: 'Plugins suggested search term' } ),
+			this.props.translate( 'Writing', { context: 'Plugins suggested search term' } ),
+		];
+
 		return (
-			<div className="plugins-browser__main-header">
-				{ this.getSearchBox( false ) }
-			</div>
+			<SectionNav selectedText={ this.props.translate( 'Suggested Searches', { context: 'Suggested searches for plugins' } ) }>
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?